modules/nf-core/bcftools/stats/tests/main.nf.test

```diff
@@ -0,0 +1,180 @@
+nextflow_process {
+
+    name "Test Process BCFTOOLS_STATS"
+    script "../main.nf"
+    process "BCFTOOLS_STATS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bcftools"
+    tag "bcftools/stats"
+
+    test("sarscov2 - vcf_gz") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                            file(params.test_data['sarscov2']['illumina']['test_vcf_gz'], checkIfExists: true),
```

Comment:
Can you update these paths according to e.g. this [PR](https://github.com/nf-core/modules/pull/5281/files)?
That way is preferred to run the tests properly on the cloud mirror (afaik)

Reply:
So we stop using this dictionary? https://github.com/nf-core/modules/blob/master/tests/config/test_data.config